seqr/views/utils/permissions_utils.py

```diff
@@ -59,19 +65,25 @@ def _map_anvil_seqr_permission(anvil_permission):
     if anvil_permission.get('pending'):
         return None
     access_level = anvil_permission.get('accessLevel')
-    if access_level in ['WRITER', 'OWNER', 'PROJECT_OWNER']:
+    if access_level in [WRITER_ACCESS_LEVEL, OWNER_ACCESS_LEVEL, PROJECT_OWNER_ACCESS_LEVEL]:
         return CAN_EDIT
     return CAN_VIEW if access_level == 'READER' else None
 
 
 def anvil_has_perm(user, permission_level, project):
     if not project_has_anvil(project):
         return False
-    workspace_permission = user_get_workspace_access_level(user, project.workspace_namespace, project.workspace_name)
+    return workspace_has_perm(user, permission_level, project.workspace_namespace, project.workspace_name)
+
+
+def workspace_has_perm(user, permission_level, namespace, name, can_share=False, can_compute=False):
+    workspace_permission = user_get_workspace_access_level(user, namespace, name)
     if not workspace_permission:
         return False
+    perm = workspace_permission.get(CAN_SHARE_PERM, False) if can_share else True
+    perm = perm and workspace_permission.get(CAN_COMPUTE_PERM, False) if can_compute else True
```

Comment:
do we ever user `can_compute`? If not we should take it out for now since its kind of confusing logic that isn't ever used

Reply:
Okay, take out `can_compute` for now.